maintainers/maintainer-list.nix

```diff
@@ -1084,6 +1084,12 @@
     githubId = 75972;
     name = "Ben Booth";
   };
+  berberman = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add berberman``.